### PR TITLE
style: first iteration in mobile support

### LINKED ISSUE #696: Mobile UI and check working flows

### PR DESCRIPTION
**Pull request by cazala**
I don't think it completely closes #696 but it's a start

moubail

I did some nasty stuff in this PR I know 🙈 🕙 🔥

### CODE REVIEW

**Comment by nachomazzara** [comment]
Just in case there is not the expected behaviour and we want to support at least iphone SE resolution:

screen shot 2018-11-28 at 18 18 51

**Comment by cazala** [comment]
@nachomazzara we've got are for iPhone 6+ (min width of 375px)

**Comment by nachomazzara** [comment]
I know it is not part of this PR, but we could fix this: 

screen shot 2018-11-28 at 18 55 59


```css
.BidConfirmationModal .modal-body .actions .button {
    max-width: 242px;
    height: 44px;
    display: block;
    min-width: 190px;
    width: 100%;
}
```

**Comment by cazala** [comment]
@nachomazzara fixed by https://github.com/decentraland/marketplace/pull/727/commits/3d06479f13e2b5b7f451345d5f562d023837ec20 👍

File: webapp/src/components/AuctionPage/AuctionPage.css [hunk 1]
```diff
@@ -82,25 +82,32 @@
...
   display: flex;
   align-items: center;
 }
 
-.AuctionPage .bid-wrapper .ui.button {
+.AuctionPage .output-information .ui.button {
   min-width: 144px;
 }
-.AuctionPage .bid-wrapper {
+.AuctionPage .output-information {
```
Comment (nachomazzara) [inline webapp/src/components/AuctionPage/AuctionPage.css]: Missing blank line here

File: webapp/src/components/AuctionPage/AuctionPage.css [hunk 2]
```diff
@@ -82,25 +82,32 @@
...
   background-color: var(--dark-gray);
   padding: 12px 16px;
   border-radius: 10px;
 }
-.AuctionPage .bid-wrapper .information-block {
+.AuctionPage .output-information .information-block {
```
Comment (nachomazzara) [inline webapp/src/components/AuctionPage/AuctionPage.css]: Missing blank line here

File: webapp/src/components/AuctionPage/AuctionPage.js [hunk 3]
```diff
@@ -304,6 +311,11 @@ export default class AuctionPage extends React.PureComponent {
       parcel => parcel.owner == null
     )
 
+    let auctionMenuClasses = 'auction-menu'
```
Comment (nachomazzara) [inline webapp/src/components/AuctionPage/AuctionPage.js]: Maybe we could move this _class generator_ to a component class

```javascript
getAuctionMenuClassNames = () => {
   let auctionMenuClasses = 'auction-menu'
   if (this.state.toggle) {
      auctionMenuClasses += ' open'
    }
   return auctionMenuClasses
}

**Pull request merged**